### DISCUSSION TITLE
Needs more readable documentation

### DISCUSSION THREAD

**Discussion by austin-convoy** [jamessimone/apex-rollup#315] [Ideas]
This is such a great, more nuanced and transaction-sensitive alternative to DLRS that it's a shame the documentation is so dense. I'm not shy to technical documentation but this is pretty difficult to get through. I think it would help if James got someone who has expertise in the package but wasn't the original author to help rewrite the README because it mostly reads as someone who A) wrote the code (unsurprisingly) and B) has a way better mental model of the architecture in their head. 

e.g. the documentation for "Rollup Control CMDT" should be in a separate section with a label like "Configure Rollup Settings with Control Metadata". Action statements are easier to understand than rote description.

**Comment by jamessimone** [comment]
@austin-convoy thanks for starting this discussion and I agree that the documentation could use a once-over.  I'm of course open to incremental improvements - which may be more palatable and approachable - to the README in particular. I'm happy to make your proposed edit to the `Rollup Control CMDT` as part of the ongoing work I have for the next release.

With many possible audiences engaging with the documentation, satisfying all parties when it comes to approachability versus detail will likely continue to be a balancing act; I say this as a word of caution when it comes to action statements being easier to understand. I appreciate your perspective and will, whenever possible, attempt to improve upon the current documentation denseness.
